### ISSUE TITLE
include citations in mission briefs

### ISSUE THREAD

**Issue by davideg** [space-ship-earth/pyspaceship#61]
It could be helpful to include links to evidence for the claims we make in our mission briefs. I'm picturing something wikipedia style.

For example:

Most Americans eat more meat protein than our bodies need. Also, eating more meat increases your risk of having a heart attack or a stroke. If you reduce the amount of meat you eat, you will help yourself be healthier in the long run.

Could become:

Most Americans eat more meat protein than our bodies need<sup><a href="https://www.hsph.harvard.edu/nutritionsource/what-should-you-eat/protein/#how-much-need">[1]</a></sup>. Also, eating more meat increases your risk of having a heart attack or a stroke<sup><a href="https://www.nih.gov/news-events/nih-research-matters/eating-red-meat-daily-triples-heart-disease-related-chemical">[2]</a></sup>. If you reduce the amount of meat you eat, you will help yourself be healthier in the long run.

[Healthline](https://www.healthline.com/nutrition/11-proven-benefits-of-bananas) could be another source of inspiration.

**Comment by svetlichnaya** [comment]
I agree & love these kinds of citations! This is a placeholder, copy & pasted from the content folder in Dropbox. We'll probably want to transfer this content into Git at some point? Easier for now for Emily & others to draft/collaborate on it outside of Git.
